### ISSUE TITLE
does not find 'make'?

### ISSUE THREAD

**Issue by EstherJohanna** [google/or-tools#393]
Hi all,

I'm trying to do the binary installation for Windows with support for python. I put the ortools_examples folder in C:\Users\user\Anaconda2\Lib\site-packages. When I navigate the Visual Studio command window to the ortools_examples folder and type 'make install', it tells me that it cannot find 'make'. Do you know what my mistake is? Do you need more info?

**Comment by maxisawesome** [comment]
I also ran into this issue on windows 10 binary installation. I think its something about windows not realizing that make is a command from the makefile and trying to run a command called "make" that doesn't actually exist. I'm fairly certain its an issue that's not specific to or-tools but rather its an issue with windows commands. Any help/corrections would be much appreciated, as I did not successfully solve the problem.
